src/app/clients/clients-view/clients-view.component.html

```diff
@@ -62,6 +62,10 @@ <h3>
 
             <br />
             <span *ngIf="clientViewData.mobileNo">Phone No: {{ clientViewData.mobileNo }} &nbsp;</span>
+
+            <span *ngIf="clientViewData.dateOfBirth">
+              | Birth Date: {{ clientViewData.dateOfBirth | dateFormat }} &nbsp;</span
```

Comment:
The | should be in the mobileNo span just in case the mobileNo is not provided, the | should not be displayed

Reply:
@amydoxym hmm, tricky, what if the mobile number is displayed but the date of birth is not present.

Reply:
Let then do this
```
 <span *ngIf="clientViewData.mobileNo">Phone No: {{ clientViewData.mobileNo }} &nbsp;</span>
<span *ngIf="clientViewData.mobileNo && clientViewData.dateOfBirth"> | </span>
<span *ngIf="clientViewData.dateOfBirth">
 Birth Date: {{ clientViewData.dateOfBirth | dateFormat }} &nbsp;</span>
```